tools/docker/builder/Dockerfile

```diff
@@ -12,7 +12,8 @@ FROM $image
 RUN apt-get update && apt-get install -y \
 	curl \
 	gcc \
-	cmake \
+        cmake \
```

Comment:
What's up with the indentation here?

Reply:
This time it's the other way around: the file is using tabs, and I used spaces in this commit.
In this case it probably makes more sense to keep on using tabs (or convert them all to spaces, but not mixing them).